components/eventing-controller/config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
+# More document can be found at https://docs.cert-manager.io
+# WARNING: Targets CertManager v1.0. Check https://cert-manager.io/docs/installation/upgrading/ for breaking changes.
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: eventing-selfsigned-issuer
+  namespace: kyma-system
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
```

Comment:
is this the certificate required by the conversion webhook ?

Reply:
Without the autogenerated certs that would be implemented with [this](https://github.com/kyma-project/kyma/issues/15346) issue, we currently need cert-manager for development.

Reply:
its for dev purposes, it will be needed until proper cert solution is implemented